.buildkite/release.yml

```diff
@@ -0,0 +1,13 @@
+agents:
+  provider: "gcp"
```

Comment:
nit: does this need to run on `gcp`?

from my experience with playing around with buildkite, this provider takes a bit longer to get scheduled than other workers.

Reply:
It is needed, since the existing default containers don't have the whole OS setup, such as `gpg`.

Potentially, we could create a container image to support that particular use case, but as it works with `GCP` workers, I'd consider that particular implementation to be done in a follow up. So we can keep migrating other projects.